deploy/kubernetes/wait-for-driver.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# This script waits for the deployments from ./deploy-driver.sh to be ready.
+
+print_usage()
+{
+    printf "wait-for-driver.sh [--windows]\n"
+    printf "\t--windows: wait on the windows deployment rather than linux"
+    echo
+}
+
+node_daemonset=csi-gce-pd-node
+while [[ -n "${1-}" ]]; do
+  case $1 in
+    --windows )    shift
+                   node_daemonset=csi-gce-pd-node-win
+                   ;;
+    -h | --help )  print_usage
+                   exit 1
+                   ;;
+    * )            print_usage
+                   exit 1
+                   ;;
+  esac
+done
+
+kubectl wait -n gce-pd-csi-driver deployment csi-gce-pd-controller --for condition=available
+
+retries=60
```

Comment:
so the max waiting time is 10 mins. I have seen for windows it might take slightly longer than 10 mins to come up. How about set it to 12 mins?

Reply:
Good point updated it to 15m